### ISSUE TITLE
Recommendations on Airdrop Requirements

### ISSUE THREAD

**Issue by Gith5champ** [scroll-tech/contribute-to-scroll#345]
### Name of the project

Scroll Interactions

### Project type

Library

### Site of the project

scroll.io

### Github repository of the project

https://github.com/scroll-tech/contribute-to-scroll

### Networks

Scroll Mainnet

### Additional context

So, as we know, the $Strk team didn't do well in the just-announced criteria/distribution of their airdrop. This was not what was expected from a project of such high expectations. Be that as it may, it's essential that we move forward and possibly provide recommended solutions to other projects in how to handle their airdrops in this airdrop season. One of such anticipated projects for the year would be Scroll.

2024 has been tagged by some as the Golden Age of Airdrops. If tokenless Projects and users get it right, there could be serious cashouts this year where everyone is happy.

Hence, in keeping with the fact that Scroll is almost certainly going to do an airdrop for users of the Chain and ecosystem, these are my recommendations for the criteria, and I hope it is looked into. (I will do my best to add the reasons for any criteria where necessary)

1. Testnet Participation: Long before Mainnet launched and users began to use funds to push transactions on the Platform for the "Airdrop", a few, selected community members were required to try out the project's platform in its raw, underdeveloped form to try out its capacity and all. It is only "sensible" to recognize those who stepped out of their way to do this. Please take note that there couldn't have been any Sybilling or Botting of the Testnet phase as users were required to fill Google forms and so, each unique Google form has a Gmail/email attached to it. This can be used, as Aptos did with their Airdrop 1, to filter Bots and Sybils for the Testnet phase.
2. Contract Deployment program: during a period of 60 days of the Genesis block, users were required to deploy contracts according to tiers and ranking. And these parameters were used to determine what kind of NFT they were eligible to mint on 14th December 2023. This is very significant because along with the Testnet participants, it allows users the opportunity to be tagged "early adopters" of the project, and as such, they are worthy of being commended with more than just words (😍)
3. Mainnet users: this can come in the form of volume transacted, the native bridge used, contracts interacted with, Source/Destination Chains, number of transactions, minimum months interacted with, minimum balance held in the chain (which should include those in LP as well), etc. Users need to know that their efforts of months count for something tangible. It is obvious that nobody uses a project "just for the tech" and not for making money.
4. Contributors in Github: The $STRK team got this spot-on, as it's been classified as a "Developers' chain" from the beginning. Those who contribute meaningful ideas to the team are also worthy of appraisal.
5. Any other criteria that the team deems fit.

With these in place, it becomes easier to make everyone happy, team, users, VCs…everyone.

Thanks.

**Comment by greghfield** [comment]
I like the idea for sybill prevention. Id like to see more pay off for those that have interacted on Galxe Scroll quests as well. Any work that contributes to scroll or any chain should be counted for something! Imo

**Comment by worldzofai** [comment]
I actually like this. Would be interesting to see how they would implement sybill prevention.

**Comment by Mindstate1988** [comment]
I like this, especially the testnet phase. I think we got Scroll OG role for that in the discord?

**Comment by vinniee13** [comment]
This is a very fair suggestion tbh.. i hope the Team considers this we wouldnt want a repeat of $STRK

**Comment by Olasolo3** [comment]
True, It is actually imperative to filter bots and Sybils for fair reward of contributors and users for the sustainability of the projects, however the importance of projects community, value and interest is gradually drifting way. I hope Scroll intend not to only consider and articulate measures and criteria in fairness but also a bring back the believe in projects transparency

**Comment by Showgit0522** [comment]
Solid suggestions. I love it

**Comment by abah123456** [comment]
Great idea
